test/models/vrp_consistency_test.rb

```diff
@@ -41,14 +41,14 @@ def test_reject_if_periodic_with_any_relation
       ['shipment', 'meetup',
        'same_route', 'sequence', 'order',
        'minimum_day_lapse', 'maximum_day_lapse', 'minimum_duration_lapse', 'maximum_duration_lapse',
-       'vehicle_group_duration', 'vehicle_group_duration_on_weeks', 'vehicle_group_duration_on_months'].each{ |relation_type|
+       'vehicle_group_duration', 'vehicle_group_duration_on_weeks', 'vehicle_group_duration_on_months', 'vehicle_trips'].each{ |relation_type|
         vrp[:relations] = [{
             type: relation_type,
             linked_ids: ['service_1', 'service_2']
         }]
 
         assert_raises OptimizerWrapper::DiscordantProblemError do
-            TestHelper.create(vrp)
+          TestHelper.create(vrp)
```

Comment:
Do we need to call the `create(vrp)` in these tests? Can't we call `check_consistency` (or the other sub-functions of vrp.create) so that we don't actually call `active_hash` and create the VRP object? Because `DiscordantProblemError` would be only raise by these sub-functions no? 

It looks like there are other tests like this -- i.e., they call create just for the raise but then don't use the generated VRP object.

Reply:
I don't repeat the same comment for the other tests down below but  I think they can all just call the apprriate "validation" function.